habitat-baselines/habitat_baselines/config/test/ddppo_instance_imagenav_test.yaml

```diff
@@ -1,4 +1,4 @@
-BASE_TASK_CONFIG_PATH: configs/tasks/instance_imagenav_hm3d.yaml
+BASE_TASK_CONFIG_PATH: habitat-lab/habitat/config/tasks/instance_imagenav_hm3d.yaml
```

Comment:
Remove `habitat-lab/habitat/config/` prefix here as in all configs that have `_test` suffix?
(or opposite - add prefix to all other test configs)